apps/namada-interface/src/utils/theme/theme.ts

```diff
@@ -333,6 +334,17 @@ export enum ThemeName {
   Placeholder,
 }
 
+export type ColorMode = "light" | "dark";
```

Comment:
I went with union type instead of enum here. There are few upsides. It does not produce new object when compiled. Potentially you can "compare" to it without importing it explicitly. You can export it as a type to avoid potential circular dependencies.